help_to_heat/utils.py

```diff
@@ -238,3 +238,11 @@ def get_current_and_next_month_names(month_names):
     current_month = month_names[now.month - 1]
     next_month = month_names[(now + relativedelta(months=+1)).month - 1]
     return current_month, next_month
+
+
+# Q1 january -> april
+# Q2 april -> july
+# Q3 july -> october
+# Q4 october -> january
+def get_current_and_next_quarter_month_names(month_names):
```

Comment:
Would like a comment explaining that we update this manually and why

Reply:
have explained a bit more, for what its worth there's a comment in the reverse direction at https://softwiretech.atlassian.net/wiki/spaces/Support/pages/20520173599/DESNZ+GBIS+Help+to+Heat+Common+Tasks#D)-How-to-load-an-EPC-data-dump-for-fun-%26-profit